src/components/checkbox/checkbox.css

```diff
@@ -0,0 +1,12 @@
+@import '../../styles/colors-v2.css';
+@import "../../styles/spacing.css";
+@import "../../styles/typography-v2.css";
+
+.input {
+  height: 20px;
```

Comment:
We want the input or its container's height to be `height: var(--spacing-x-large);`. I am digging the thicker checkbox but I am going to guess design won't dig it?

Instead, let's make its container 32px, leave native sizing of the checkbox, and align center. Something like:

![image](https://user-images.githubusercontent.com/841084/116919266-e1521900-ac05-11eb-83ca-ef08b1aafc1a.png)

I talked with Anne about it and we didn't come to a final design for a checkbox field. We'll probably iterate a bit after the fact.

Reply:
db5d72ea5b088457688e7001e039823573617265

I didn't add the align centering because it didn't seem needed, but the container is the right size now.